src/plot_api/validate.js

```diff
@@ -397,6 +397,8 @@ function isInSchema(schema, key) {
 }
 
 function getNestedSchema(schema, key) {
+    if(key in schema) return schema[key];
+
```

Comment:
Not part of templating, just noticed it as there's similar logic in templates and I was looking to see how you handled it here 🙈